From 8b8c45b9fdb5297fe2b4c8ebfd61916ab5da5e48 Mon Sep 17 00:00:00 2001
From: Joel A Fernandes <joelagnel@ti.com>
Date: Thu, 25 Oct 2012 00:41:59 -0500
Subject: [PATCH 5/9] Codec sits on i2c3 bus when cape is connected to
 beaglebone LT. As a part of getting audio working on
 beaglebone LT, we first try with cape.

This patch adds i2c3 node and codec slave address.

Adapted from following patch by Gururaja Hebbar <gururaja.hebbar@ti.com>:

    On am335x-boneblack, codec TLV320AIC3x sits on i2c1 bus.
    This patch also add I2C1 node with I2C frequency and TLV320AIC3x Codec
    I2C slave address.

Signed-off-by: Joel A Fernandes <joelagnel@ti.com>
---
 arch/arm/boot/dts/am335x-boneblack.dts |   12 ++++++++++++
 1 files changed, 12 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/am335x-boneblack.dts b/arch/arm/boot/dts/am335x-boneblack.dts
index fdbaa75..987291b 100644
--- a/arch/arm/boot/dts/am335x-boneblack.dts
+++ b/arch/arm/boot/dts/am335x-boneblack.dts
@@ -53,6 +53,18 @@
 	};
 };
 
+&i2c3 {
+	status = "okay";
+	clock-frequency = <400000>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&i2c3_pins>;
+ 
+	tlv320aic3x: tlv320aic3x@1b {
+		compatible = "ti,tlv320aic3x";
+		reg = <0x1b>;
+	};
+};
+
 &userled_pins {
 	pinctrl-single,pins = <
 		0x54 0x7	/* gpmc_a5.gpio1_21, OUTPUT | MODE7 */
-- 
1.7.7.6

